source/open_node/ast_visitor/ast_visitor_examples.py

```diff
@@ -0,0 +1,71 @@
+from open_node.ast import (
+    Identifier,
+    QuantumGateModifier,
+    QubitDeclaration,
+    QuantumGate,
+    QuantumGateDefinition,
+)
+from open_node.ast_visitor.ast_visitor import NodeVisitor, NodeTransformer
+
+
+class OpenNodePrinter(NodeVisitor):
+    """
+    Example visitor which "unparses" the Program back to text
```

Comment:
Should probably point out that this is a partial implementation

Reply:
@shiyunon iirc you had made more progress on a printer?

Reply:
Yes, I have most of it done. @aspcompiler and I discussed and decided to not include it in this initial PR for (1) easier review (2) making the decision of putting it in `ast.py` vs another file public and more open.

Reply:
OK. Since ast_visitor_examples.py have 2 incomplete examples that you recommend to remove, I will remove the file for now and add back in future PRs when we complete.